docs/user-guide/api-mediation/configuration-saf-resource-checking.md

```diff
@@ -10,44 +10,47 @@ API ML can check for the authorization of the user on certain endpoints. Access
 
 Verification of the SAF resource is provided by the following three providers:
 
-- **`endpoint`**  
-This is the highest priority provider, such as a REST endpoint call (ZSS or similar one). This option is disabled by default. In Zowe, ZSS has the API to check for SAF   resource authorization.
-  
 - **`native`**  
-The Native JZOS classes from Java are used to determine SAF resource access. This is the default provider.
-  
+  The Native JZOS classes from Java are used to determine SAF resource access. This is the default provider.
+
+  **Note:** This provider cannot be used off-platform.
+
+- **`endpoint`**  
+The Endpoint provider relies on APIs such as a REST endpoint call (ZSS or similar one). This option is disabled by default. In Zowe, ZSS provides the API to check for SAF resource authorization.
+
 - **`dummy`**  
 This is the lowest priority provider. This is the dummy implementation and is defined in a file.
 
 :::note
-Verification of the SAF resource uses the first available provider based on the specified priority. The default configuration resolves to the native provider. 
+Verification of the SAF resource uses the first available provider based on the specified priority. The default configuration resolves to the `native` provider. 
 :::
 
-You can select a specific provider by specifying the `components.gateway.apiml.security.authorization.provider` key in the `zowe.yaml` file. Use the parameter value to
+You can select a specific provider by specifying the `components.gateway.apiml.security.authorization.provider` key in the `zowe.yaml` file. 
+If no value is assigned to Use the parameter value to
 strictly define a provider. If verification is disabled, select the `endpoint` option. 
```

Comment:
I think this needs to be rewritten

Reply:
I've fixed that, thanks. I'm just not totally sure about `If verification is disabled, select the `endpoint` option.` meaning